src/Illuminate/Database/Schema/Blueprint.php

```diff
@@ -731,9 +731,32 @@ public function foreign($columns, $name = null)
      */
     public function id($column = 'id')
     {
+        if (Builder::$defaultIdType === 'uuid') {
+           return $this->uuid($column)->primary();
+        } elseif (Builder::$defaultIdType === 'ulid') {
+          return  $this->ulid($column)->primary();
```

Comment:
Your indention is off here
```suggestion
            return $this->uuid($column)->primary();
        } elseif (Builder::$defaultIdType === 'ulid') {
            return  $this->ulid($column)->primary();
```